packages/core/src/browser/keybinding.ts

```diff
@@ -279,23 +339,43 @@ export class KeybindingRegistry {
         return result;
     }
 
+    /**
+     * Returns wether a binding is being shadowed or not inside scope
```

Comment:
s/wether/whether, unless you really mean [a castrated ram](http://www.grammar-monster.com/easily_confused/weather_whether_wether.htm) :)

Reply:
Done.